### ISSUE TITLE
Footer JS-based responsiveness on pages without the sidebar is unnecessary

### ISSUE THREAD

**Issue by obulat** [WordPress/openverse#5196]
## Current Situation

The footer layout needs to account for the sidebar width (if any) to make its layout responsive. This is achieved by using the JS-based responsiveness. However, this is not necessary on pages that do not have a sidebar, and lead to a layout shifts on the homepage.

## Suggested Improvement

Audit the footer layout responsiveness, and only add it on the search layout with the sidebar.

## Benefit

Improved performance and less layout shifting.

**Comment by dhruvkb** [comment]
@obulat instead of JS-based responsiveness on pages with a sidebar and regular media queries in pages without a sidebar, i think using [container queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_containment/Container_queries) consistently in both scenarios would be the better approach.

**Comment by obulat** [comment]
I agree, @dhruvkb. This looks like a good guide on implementing container queries while also supporting older browsers: https://web.dev/blog/how-to-use-container-queries-now 
We'll need to create a solution that works with Tailwind

**Comment by dhruvkb** [comment]
Tailwind supports container queries [with an official plugin](https://tailwindcss.com/blog/tailwindcss-v3-2#container-queries). I hadn't considered that older browsers will need workarounds, thanks for the guide!
